source/extensions/extensions_build_config.bzl

```diff
@@ -3,363 +3,332 @@ EXTENSIONS = {
     #
     # Access loggers
     #
-
-    "envoy.access_loggers.file":                        "//source/extensions/access_loggers/file:config",
-    "envoy.access_loggers.extension_filters.cel":       "//source/extensions/access_loggers/filters/cel:config",
-    "envoy.access_loggers.http_grpc":                   "//source/extensions/access_loggers/grpc:http_config",
-    "envoy.access_loggers.tcp_grpc":                    "//source/extensions/access_loggers/grpc:tcp_config",
-    "envoy.access_loggers.open_telemetry":              "//source/extensions/access_loggers/open_telemetry:config",
-    "envoy.access_loggers.stdout":                      "//source/extensions/access_loggers/stream:config",
-    "envoy.access_loggers.stderr":                      "//source/extensions/access_loggers/stream:config",
-    "envoy.access_loggers.wasm":                        "//source/extensions/access_loggers/wasm:config",
+    "envoy.access_loggers.file": "//source/extensions/access_loggers/file:config",
```

Comment:
There are lots of changes to this file which seem unexpected. Can you revert them so the file only contains changes that are part of this PR?

Reply:
Noted will change and resolve. Waiting for validations to complete